webapp/templates/webapp/recipient/detail.html

```diff
@@ -13,6 +13,13 @@
   {% endfor %}
   </dd>
 </dl>
+{% for group in object_list %}
```

Comment:
@migonzalvar He probado estas tres posibilidades, solo muestra no hay grupos(los grupos los he creado a mano en el admin,es decir hay grupos creados)
{% for group in group.group_name_set.all %}
{% for group in group.group_name %}